packages/filesystem/src/browser/file-tree/file-tree.ts

```diff
@@ -136,11 +199,32 @@ export namespace DirNode {
         };
     }
 
+    export function createWorkspaceRoot(workspace: FileStat, children: TreeNode[]): DirNode {
```

Comment:
Don't we agree that `FileTree` remains agnostic to workspace concepts since `@theia/filesystem` extension is not aware of them and we do changes `NavigatorFileTree` instead?